arcgis-ios-sdk-samples/Scenes/Surface placements/SurfacePlacementsViewController.swift

```diff
@@ -40,39 +40,39 @@ class SurfacePlacementsViewController: UIViewController {
+        self.absoluteGraphicsOverlay.sceneProperties?.surfacePlacement = .absolute
         
         //add graphic overlays to the scene view
-        self.sceneView.graphicsOverlays.addObjectsFromArray([self.drapedGraphicsOverlay, self.relativeGraphicsOverlay, self.absoluteGraphicsOverlay])
+        self.sceneView.graphicsOverlays.addObjects(from: [self.drapedGraphicsOverlay, self.relativeGraphicsOverlay, self.absoluteGraphicsOverlay])
         
         //add graphics
         self.addGraphics()
     }
     
     
-    private func addGraphics() {
+    fileprivate func addGraphics() {
         
         //create point for graphic location
-        let point = AGSPoint(x: -4.04, y: 53.06, z: 1000, spatialReference: AGSSpatialReference.WGS84())
+        let point = AGSPoint(x: -4.04, y: 53.06, z: 1000, spatialReference: AGSSpatialReference.wgs84())
         
-        self.drapedGraphicsOverlay.graphics.addObjectsFromArray([AGSGraphic(geometry: point, symbol: self.pointSymbol(), attributes: nil), AGSGraphic(geometry: point, symbol: self.textSymbol("Draped"), attributes: nil)])
-        self.relativeGraphicsOverlay.graphics.addObjectsFromArray([AGSGraphic(geometry: point, symbol: self.pointSymbol(), attributes: nil), AGSGraphic(geometry: point, symbol: self.textSymbol("Relative"), attributes: nil)])
-        self.absoluteGraphicsOverlay.graphics.addObjectsFromArray([AGSGraphic(geometry: point, symbol: self.pointSymbol(), attributes: nil), AGSGraphic(geometry: point, symbol: self.textSymbol("Absolute"), attributes: nil)])
+        self.drapedGraphicsOverlay.graphics.addObjects(from: [AGSGraphic(geometry: point, symbol: self.pointSymbol(), attributes: nil), AGSGraphic(geometry: point, symbol: self.textSymbol("Draped"), attributes: nil)])
+        self.relativeGraphicsOverlay.graphics.addObjects(from: [AGSGraphic(geometry: point, symbol: self.pointSymbol(), attributes: nil), AGSGraphic(geometry: point, symbol: self.textSymbol("Relative"), attributes: nil)])
+        self.absoluteGraphicsOverlay.graphics.addObjects(from: [AGSGraphic(geometry: point, symbol: self.pointSymbol(), attributes: nil), AGSGraphic(geometry: point, symbol: self.textSymbol("Absolute"), attributes: nil)])
     }
     
-    private func pointSymbol() -> AGSSimpleMarkerSceneSymbol {
-        return AGSSimpleMarkerSceneSymbol(style: .Sphere, color: UIColor.redColor(), height: 50, width: 50, depth: 50, anchorPosition: .Center)
+    fileprivate func pointSymbol() -> AGSSimpleMarkerSceneSymbol {
+        return AGSSimpleMarkerSceneSymbol(style: .sphere, color: UIColor.red, height: 50, width: 50, depth: 50, anchorPosition: .center)
     }
     
-    private func textSymbol(text: String) -> AGSTextSymbol {
-        return AGSTextSymbol(text: text, color: UIColor.blueColor(), size: 20, horizontalAlignment: .Left, verticalAlignment: .Middle)
+    fileprivate func textSymbol(_ text: String) -> AGSTextSymbol {
```

Comment:
similar to other comments above about having argument name to clarify call site usage